lacework/resource_lacework_vulnerability_exception_container.go

```diff
@@ -322,7 +344,15 @@ func resourceLaceworkVulnerabilityExceptionContainerRead(d *schema.ResourceData,
 	vulnCriteria["severities"] = response.Data.VulnerabilityCriteria.Severity
 	vulnCriteria["cves"] = response.Data.VulnerabilityCriteria.Cve
 	vulnCriteria["package"] = vulnerabilityExceptionFlattenPackages(response.Data.VulnerabilityCriteria.Package)
-	vulnCriteria["fixable"] = response.Data.VulnerabilityCriteria.FixableEnabled()
+
+	if response.Data.VulnerabilityCriteria.FixableEnabled() != nil {
+		if _, ok := d.GetOk("vulnerability_criteria.0.fixable"); ok {
+			vulnCriteria["fixable"] = response.Data.VulnerabilityCriteria.FixableEnabled()
+		} else {
+			vulnCriteria["fixable_vuln"] = response.Data.VulnerabilityCriteria.FixableEnabled()
```

Comment:
This should be converted to a string? I mean, `FixableEnabled()` returns a boolean pointer so 🤔